src/zip.js

```diff
@@ -85,16 +80,15 @@ const zipFunction = async function (
     return
   }
 
-  const { extension, filename, mainFile, runtime, srcDir, stat } = functions.values().next().value
+  const { config, extension, filename, mainFile, runtime, srcDir, stat } = functions.values().next().value
   const pluginsModulesPath =
     defaultModulesPath === undefined ? await getPluginsModulesPath(srcPath) : defaultModulesPath
 
   await makeDir(destFolder)
 
   const zipResult = await runtime.zipFunction({
+    config,
```

Comment:
We should make a major release for this PR since it is technically a breaking change.
Should we release #373 at the same time?

Reply:
Yes, good call!

Reply:
I updated the title of the PR to add `!` for conventional commits.